pages/design/rule-design.md

```diff
@@ -163,7 +163,7 @@ Applicability and expectations, often have multiple conditions that need to be m
 
 This phrasing is designed to be easily readable, but may not work in every situation. In all cases prefer readability over prescriptive formatting. Additionally to this, keep the following in mind when listing conditions:
 
-- The phrasing "for which (all | one of) the following is" can be modified based on use, such as "to achieve one of the following objectives".
+- The phrasing "for which (all | at least one of) the following is" can be modified based on use, such as "to achieve at least one of the following objectives".
```

Comment:
Should we mention here the "except if one or more of the following" construction?